src/test/java/hudson/plugins/mercurial/ChangeComparatorTest.java

```diff
@@ -56,7 +57,7 @@ public Change compare(MercurialSCM scm, Launcher launcher, TaskListener listener
 				listener, 
 				new MercurialTagAction("tip","",null,null),
 				listener.getLogger(), 
-				Computer.currentComputer().getNode(), 
+				Jenkins.getInstance().toComputer().getNode(), 
```

Comment:
Yes, and the `.toComputer().getNode()` is a gratuitous round-trip, since the `Jenkins` instance was the `Node` you are getting back anyway.